website/content/docs/internals/telemetry/metrics/availability.mdx

```diff
@@ -0,0 +1,118 @@
+
+@include 'telemetry-metrics/vault/replication/rpc/client/forward.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/client/guard_hash.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/client/persist_alias.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/client/register_auth.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/client/register_lease.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/client/save_mfa_response_auth.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/client/stream_wals.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/client/sub_page_hashes.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/client/sync_counter.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/client/upsert_group.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/client/wrap_in_cubbyhole.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/dr/server/echo.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/dr/server/fetch_keys_request.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/server/auth_request.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/server/bootstrap_request.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/server/conflicting_pages_request.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/server/echo.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/server/forwarding_request.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/server/guard_hash_request.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/server/persist_alias_request.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/server/persist_persona_request.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/server/save_mfa_response_auth.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/server/stream_wals_request.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/server/sub_page_hashes_request.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/server/sync_counter_request.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/server/upsert_group_request.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/standby/server/create_token_register_auth_lease_request.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/standby/server/echo.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/standby/server/register_auth_request.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/standby/server/register_lease_request.mdx'
+
+@include 'telemetry-metrics/vault/replication/rpc/standby/server/wrap_token_request.mdx'
+
+@include 'telemetry-metrics/vault/replication/wal/gc.mdx'
+
+@include 'telemetry-metrics/vault/replication/wal/last_dr_wal.mdx'
+
+@include 'telemetry-metrics/vault/replication/wal/last_performance_wal.mdx'
+
+@include 'telemetry-metrics/vault/replication/wal/last_wal.mdx'
+
+## Rollback metrics
+
+@include 'telemetry-metrics/rollback-intro.mdx'
```

Comment:
I don't think rollback belongs here, but I understand if you dumped it here because you couldn't find anywhere else to group it.

Reply:
Maybe core-system.mdx.

Reply:
Oh it's already there, I suggest just removing it from here then.

Reply:
Yeah, since they're partials, we can include them wherever it makes sense that someone might look to find them. I think I conflated fallback and rollback in my brain, which is why I added it to the availability index. I'll remove it.